src/Illuminate/Foundation/Http/Middleware/VerifyCsrfToken.php

```diff
@@ -7,9 +7,12 @@
 use Symfony\Component\HttpFoundation\Cookie;
 use Illuminate\Contracts\Encryption\Encrypter;
 use Illuminate\Session\TokenMismatchException;
+use Illuminate\Http\Middleware\PassThroughTrait;
 
 class VerifyCsrfToken
 {
+    use PassThroughTrait;
```

Comment:
I would also include it to the others middleware, so that we let this passthrough choice for the other ones, especially the `\Illuminate\Session\Middleware\StartSession` one ?